AK/HashMap.h

```diff
@@ -90,6 +90,18 @@ class HashMap {
         return m_table.find(hash, predicate);
     }
 
```

Comment:
The commit message is no longer accurate, perhaps "Allow arbitrary key types in HashTable::{find,contains}"?